src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
+
+  propTypes: {
+    data: React.PropTypes.object,
+    onChange: React.PropTypes.func,
+    selectedItem: React.PropTypes.string,
```

Comment:
@cerinman @bsbeeks 
Yes, agreed. How about we change the data structure as follows:
```javascript
[
  {
    id: 1,
    value: 'groceries',
    items: [
      {
        id: 11,
        value: 'item1'
      },
      {
        id: 12,
        value: 'item2'
      }
    ]
  },
  {
    id: 2,
    value: 'books',
    items: [
      {
        id: 21,
        value: 'book1'
      }
    ]
  }
]
```

And then return an object onSelect event